docs/apis-tools/go-client/index.md

```diff
@@ -1,10 +1,16 @@
 ---
 id: index
-title: "Go client"
+title: "Go client (Deprecated)"
 sidebar_label: "Quick reference"
 description: "Instantiate the client by passing in the address of the cluster you want to connect to in a Go application to interact with Camunda 8."
 ---
 
+:::caution
```

Comment:
Should this caution go at the top of each page in this Go section? Users might not come to this first page, and could miss the warning if this is the case?

Reply:
good point, will do that!

Reply:
https://github.com/camunda/camunda-docs/pull/4264/commits/a5a4123c6a766ce4c6a50aa22c23c911668038a8